e2e_tests/docker_image/import-old-e2e-tests.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+
+# Copyright 2021 Contributors to the Parsec project.
+# SPDX-License-Identifier: Apache-2.0
+
+# These commands are made to import the oldest version of the end-to-end tests to check that they
+# still work with the current version of the Parsec service.
+
+set -xeuf -o pipefail
+
+git clone https://github.com/parallaxsecond/parsec.git
+cd parsec
+# This commit is the oldest one which still works with current Parsec version.
+# It works with the Rust client version 0.6.0
+git checkout 2fee72fc64871472edf141906bf7f55bd59a2f8d
```

Comment:
Maybe we should tag that commit somehow - `client-stability-anchor` or smth? ⚓

Reply:
It's a good idea but I'm worried that the tag would be on the Parsec repository, next to all the release tags :/ Ideally it should be on the Rust client that is used

Reply:
Yeah, fair, though it's only a tag, not a Release.